rtgui/cropwindow.cc

```diff
@@ -93,31 +93,22 @@ CropWindow::~CropWindow ()
 
 void CropWindow::initZoomSteps()
 {
-    zoomSteps.push_back(ZoomStep("  1%", 0.01, 999));
-    zoomSteps.push_back(ZoomStep("  2%", 0.02, 500));
-    zoomSteps.push_back(ZoomStep("  5%", 0.05, 200));
-    zoomSteps.push_back(ZoomStep("  6%", 1.0/15.0, 150));
-    zoomSteps.push_back(ZoomStep("  8%", 1.0/12.0, 120));
+    zoomSteps.push_back(ZoomStep("  1%", 0.01, 999, true));
+    zoomSteps.push_back(ZoomStep("  2%", 0.02, 500, true));
+    zoomSteps.push_back(ZoomStep("  5%", 0.05, 200, true));
+    zoomSteps.push_back(ZoomStep("  6%", 1.0/15.0, 150, true));
+    zoomSteps.push_back(ZoomStep("  8%", 1.0/12.0, 120, true));
     char lbl[64];
-    for (int s = 100; s >= 50; s -= 5) {
+    for (int s = 100; s >= 11; --s) {
         float z = 10./float(s);
         sprintf(lbl, "% 2d%%", int(z * 100));
-        zoomSteps.push_back(ZoomStep(lbl, z, s));
-    }
-    for (int s = 45; s >= 27; s -= 3) {
-        float z = 10./float(s);
-        sprintf(lbl, "% 2d%%", int(z * 100));
-        zoomSteps.push_back(ZoomStep(lbl, z, s));
-    }
-    for (int s = 25; s >= 11; --s) {
-        float z = 10./float(s);
-        sprintf(lbl, "% 2d%%", int(z * 100));
-        zoomSteps.push_back(ZoomStep(lbl, z, s));
+        bool is_major = (s == s/10 * 10);
```

Comment:
@agriggio Sure, this will work when the compiler optimizes? Shouldn't that be a modulo?